docs/pages/foundations/motion/guidelines.tsx

```diff
@@ -0,0 +1,285 @@
+            text={`
+*   **Enter vs. exit transitions** (enter should be longer than exit)
+
+    * Shorter durations are applied to transitions that exit, dismiss, or collapse an element, as they require less user attention due to being swift. 
+
+    * On the other hand, enter transitions use longer durations, helping users in directing their attention on new elements appearing on the screen.
+`}
+          />
+        </Box>
+      </MainSection>
+
+      <MainSection
+        description={`
+        Animations should never cause a user harm or distract from a necessary task. However, inaccessible animations have the potential to cause seizures, motion sickness, or anxiety in users, so we must ensure we are creating accessible animations every time.
+
+        Below are 3 key considerations for accessibility. Note: these do not apply for “essential” animations, where [essential is defined](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-pause.html#essentialdef) as any animation where “if removed, would fundamentally change the information or functionality of the content, _and_ information and functionality cannot be achieved in another way that would conform.”
+`}
+        name="Accessibility"
+      >
+        <MainSection.Subsection
+          columns={2}
+          description={`
+          Ensure that any animation that is moving, blinking, scrolling or auto-playing and lasts longer than 5 seconds has an affordance for pausing and resuming the animation or an option to hide the animation.
+
+          _Looping animations should be avoided whenever possible, as they can become distracting very quickly._
+
+          Further reading: [WCAG 2.0 Pause, Stop Hide requirements](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-pause.html)
+          `}
+          title="Looping animations"
+        />
+        <MainSection.Subsection
+          columns={2}
+          description={`
+          Ensure that there are no flashing animations present, where flashing is defined as anything that flashes _more than three times in a one second period_.
+
+          Further reading: [WCAG 2.0 Three Flashes requirements](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-three-times.html)
+          `}
+          title="Flashing animations"
+        />
+        <MainSection.Subsection
+          columns={2}
+          description={`
+          Any non-essential animations should automatically be disabled when a user enables the reduced motion setting on their device. For Web, we have provided a [useReducedMotion hook](https://gestalt.pinterest.systems/web/utilities/usereducedmotion) to make this easy to implement.
+
+          Further reading for Web: [MDN prefers-reduced-motion guidelines](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion)
+          Further reading for iOS: [Apple developer guidelines](https://developer.apple.com/documentation/uikit/uiaccessibility/1615133-isreducemotionenabled)
```

Comment:
Here's some good guidelines for Android: https://appt.org/en/docs/android/samples/reduced-animations 

Here's some better guidelines for iOS: https://appt.org/en/docs/ios/samples/reduced-animations